scapy/layers/tls/cert.py

```diff
@@ -671,7 +671,7 @@ def encrypt(self, msg, t="pkcs", h="sha256", mgf=None, L=None):
         return self.pubKey.encrypt(msg, t, h, mgf, L)
 
     def verify(self, msg, sig, t="pkcs", h="sha256", mgf=None, L=None):
-        return self.pubKey.verify(msg, sig, t, h, mgf, L)
+        return self.pubKey.verify(msg, sig, t=t, h=h, mgf=mgf, L=L)
```

Comment:
Changing this seems to be sufficient, at least in my case :)

Reply:
Could you please change both anyways ? It's better that way

Reply:
Sure, shall I do the same for `encrypt` methods. They seem to have the same pattern.

Reply:
I've updated calls of `verify`, `encrypt` and `sign`.